package.json

```diff
@@ -22,6 +22,7 @@
     "start": "start-storybook -p 9001 -c storybook -s ./storybook/static",
     "build": "lerna run --parallel build -- --env production",
     "build:storybook": "build-storybook -c storybook -o ./storybook/public -s ./storybook/static",
+    "clean": "lerna run clean --parallel && lerna clean --yes && rimraf node_modules",
```

Comment:
Rather than have to modify each sub package I think having this root clean command do all cleaning would be fine.

I believe `rimraf` takes a glob pattern such that we could do : `rimraf packages/*/{dist,node_modules} node_modules` and that should clean up everything.

Reply:
👍